btree/btree.h

```diff
@@ -153,9 +153,9 @@ void m0_btree_get(struct m0_btree *arbor, const struct m0_btree_key *key,
 void m0_btree_put(struct m0_btree *arbor, struct m0_be_tx *tx,
 		  const struct m0_btree_rec *rec, const struct m0_btree_cb *cb,
 		  uint64_t flags, struct m0_btree_op *bop);
-void m0_btree_del(struct m0_btree *arbor, const struct m0_btree_key *key,
-		  const struct m0_btree_cb *cb, uint64_t flags,
-		  struct m0_btree_op *bop);
+void m0_btree_del(struct m0_btree *arbor,  struct m0_be_tx *tx,
```

Comment:
Move the tx as the last parameter since the rest of the parameters concern the node and the tree while tx is a separate operation which has nothing to do with btree really.

Reply:
okay